src/Data/OpenApi/Internal/AesonUtils.hs

```diff
@@ -268,6 +269,24 @@ sopSwaggerGenericToEncoding x =
     proxy = Proxy :: Proxy a
     opts  = swaggerAesonOptions proxy
 
+sopSwaggerGenericToEncodingWithOpts
+    :: forall a xs.
+        ( HasDatatypeInfo a
+        , HasSwaggerAesonOptions a
+        , All2 ToJSON (Code a)
+        , All2 Eq (Code a)
+        , Code a ~ '[xs]
+        )
+    => SwaggerAesonOptions
+    -> a
+    -> Encoding
+sopSwaggerGenericToEncodingWithOpts opts x =
+    let ps = sopSwaggerGenericToEncoding' opts (from x) (datatypeInfo proxy) defs
+    in pairs (pairsToSeries (opts ^. saoAdditionalPairs) <> ps)
+  where
+    proxy = Proxy :: Proxy a
+    defs = hcpure (Proxy :: Proxy AesonDefaultValue) defaultValue
```

Comment:
why is this different from `(aesonDefaults proxy)` in `sopSwaggerGenericToEncoding`?